R/zchunk_L201.en_nonco2.R

```diff
@@ -0,0 +1,358 @@
+#' module_emissions_L201.en_nonco2
+#'
+#' Generate non-CO2 emissions: pollutants, GHGs, non-CO2, BC/OCs, and reduction data.
+#'
+#' @param command API command to execute
+#' @param ... other optional parameters, depending on command
+#' @return Depends on \code{command}: either a vector of required inputs,
+#' a vector of output names, or (if \code{command} is "MAKE") all
+#' the generated outputs: \code{L201.en_pol_emissions}, \code{L201.en_ghg_emissions}, \code{L201.en_bcoc_emissions}, \code{L201.nonghg_max_reduction}, \code{L201.nonghg_steepness}, \code{L201.nonghg_max_reduction_res}, \code{L201.nonghg_steepness_res}, \code{L201.nonghg_res}, \code{L201.ghg_res}. The corresponding file in the
+#' original data system was \code{L201.en_nonco2.R} (emissions level2).
+#' @details Describe in detail what this chunk does.
```

Comment:
This file sets up all of the inputs needed for the energy system non-CO2 emissions in GCAM. This includes historical emissions, drivers (input or output), and pollution controls.